community.md

```diff
@@ -620,7 +620,7 @@ the added concern of project overreliance on individual unpaid
 volunteers, who suffer high burnout rates.  Of course, commercial
 participation can bring its own issues as noted below, including the
 potential to being prone to changing priorities more quickly than
-non-commercial participants.  In fact, at least one open source
+non-commercial participants.  In fact, at least one open-source
```

Comment:
Aha!  This was one that I deliberately left unhyphenated, just for the record, but I'm fine letting your hyphenation stand.  It's a judgement call.  One thing I learned in doing this change is that not all adjectives are the same creature (probably something linguists knew already).  In this case, the stewardship organizations themselves are not "open-source"... that is, while open-source software *is* open source (or "open-source"), open source stewardship organizations are *about* open source, which is a very different kind of attachment to the adjective.

(Well, I guess this is just what my commit message for commit c7109a202017d is saying.)

Anyway, I'm only making this comment to say that these are the sorts of not-clear-cut situations that arise when we try to lexically distinguish between noun-as-noun and the-same-noun-acting-as-adjective.  Shrug :-).